ui/src/icons/pmm/check.tsx

```diff
@@ -0,0 +1,24 @@
+import { SvgIcon, SvgIconProps } from '@mui/material';
+
+export const CheckIcon = (props: SvgIconProps) => (
+  <SvgIcon
+    width="18"
+    height="18"
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="https://www.w3.org/2000/svg"
+    {...props}
+  >
+    <g clip-path="url(#clip0)">
```

Comment:
nice approach, will keep that in mind

Reply:
Icons were taken from https://github.com/percona/pmm-doc/blob/main/variables.yml